examples/assets_machine_learning/machine_learning_example/README.md

```diff
@@ -0,0 +1,23 @@
+# machine_learning_example
```

Comment:
To be more consistent with our other examples, it would be good to replace this with a more human-readable name.